wiki/Client/Beatmap_editor/Song_setup/en.md

```diff
@@ -1,238 +1,121 @@
-
-Higher approach rates mean that hit objects will be shown for a shorter period of time, thus giving less reaction time to plan ahead. Inversely, lower approach rates mean that hit objects will be shown on the screen for a longer time, allowing more time to react, but can result in an excessive amount of hit objects appearing on the screen at once if the AR is too low.
-
----
-
-The duration of a hit object that stays visible on the screen (without mods) ranges from 1800ms at AR0 to 450ms at AR10.
-
-There are four mods that alters the approach rate timing when activated:
-
-- [Easy](/wiki/Gameplay/Game_modifier/Easy): Halves the AR value. (e.g. AR10→AR5)
-- [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock): Multiplies the AR value by 1.40, up to a maximum of AR10. (e.g. AR3→AR4.2 or AR9→AR10)
-- [Double Time](/wiki/Gameplay/Game_modifier/Double_Time): The AR value is not affected, but due to the 50% play speed increase, this causes circles to stay on screen 33% milliseconds shorter.
-- [Half Time](/wiki/Gameplay/Game_modifier/Half_Time): The AR value is not affected, but due to the 25% play speed decrease, this causes circles to stay on screen 33% milliseconds longer.
-
-While Half Time and Double Time does not change the actual given AR value, the speed difference leads to an apparent AR change, as low as AR-5 or as high as AR11. In the AR chart provided, those apparent values are given to allow easy comparison between approach speeds with and without mods. Actual AR, however, is always a number between 0 and 10. Note that AR levels scale by 120ms for below AR5 and 150ms for above AR5.
+osu!taiko and osu!mania are not affected by the approach rate changes — instead, both modes use scroll speed as alternative, which is based on the song's [tempo](/wiki/Music_theory/Tempo) and [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity).
 
 ### Overall difficulty
 
-*Main page: [Overall difficulty](/wiki/Beatmap/Overall_difficulty)*
-
-![](/wiki/shared/ODTable.png "Comparison of hit windows for different combinations of OD and game modifiers. For Half Time and Double Time combinations, the shown OD values are only valid for the hit windows of 300s, and would be different for 100s and 50s.")
+*Main page: [Overall difficulty](/wiki/Beatmap/Overall_difficulty)*\
+*Note: On the beatmap page, overall difficulty is listed as Accuracy.*
 
-*Note: The above table applies for the osu! game mode only.*\
-*Notice: On the beatmap page, Overall Difficulty is listed as Accuracy.*
+Overall difficulty (OD) is responsible for the sizes of hit windows, which define how difficult it is to achieve high accuracy. Higher OD values mean shorter timing windows requiring more accuracy and precision. Since accuracy is important for gaining HP, overall difficulty indirectly influences how hard it is to pass the beatmap.
```

Comment:
```suggestion
Overall difficulty (OD) is responsible for the sizes of hit windows, which define how difficult it is to achieve high accuracy. Higher OD values mean shorter timing windows, requiring more accuracy and precision. Since accuracy is important for gaining HP, overall difficulty indirectly influences how hard it is to pass the beatmap.
```